basic-features.md

```diff
@@ -4,77 +4,127 @@ summary: Learn about the basic features of TiDB.
+| **Indexing and constraints**                                                         | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Expression Indexes                                                                   | Experimental | Experimental | N   | N   | N   |
+| [Columnar Storage (TiFlash)](/tiflash/tiflash-overview.md)                           | Y     | Y   | Y   | N   | N   |
+| [RocksDB Engine](/storage-engine/rocksdb-overview.md)                                | Y     | Y   | Y   | Y   | Y   |
+| [Titan Plugin](/storage-engine/titan-overview.md)                                    | Y     | Y   | Experimental | Experimental | Experimental |
+| [Invisible Indexes](/sql-statements/sql-statement-add-index.md)                      | Y     | N   | N   | N   | N   |
+| [Composite `PRIMARY KEY`](/constraints.md)                                           | Y     | Y   | Y   | Y   | Y   |
+| [Unique indexes](/constraints.md)                                                    | Y     | Y   | Y   | Y   | Y   |
+| [Clustered index on integer `PRIMARY KEY`](/constraints.md)                          | Y     | Y   | Y   | Y   | Y   |
+| [Clustered index on composite or non-integer key](/constraints.md)                   | Experimental | N | N | N | N  |
+| **SQL statements** [2]                                                               | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Basic `SELECT`, `INSERT`, `UPDATE`, `DELETE`, `REPLACE`                              | Y     | Y   | Y   | Y   | Y   |
+| `INSERT ON DUPLICATE KEY UPDATE`                                                     | Y     | Y   | Y   | Y   | Y   |
+| `LOAD DATA INFILE`                                                                   | Y     | Y   | Y   | Y   | Y   |
+| `SELECT INTO OUTFILE`                                                                | Y     | Y   | N   | N   | N   |
+| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                             | Y     | Y   | Y   | Y   | Y   |
+| `UNION`, `UNION ALL`                                                                 | Y     | Y   | Y   | Y   | Y   |
+| `GROUP BY`, `ORDER BY`                                                               | Y     | Y   | Y   | Y   | Y   |
+| `START TRANSACTION`, `COMMIT`, `ROLLBACK`                                            | Y     | Y   | Y   | Y   | Y   |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md)                                | Y     | Y   | Y   | Y   | Y   |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md)                | Y     | Y   | Y   | Y   | Y   |
+| **Advanced SQL Features**                                                            | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Prepared statement cache                                                             | Y     | Experimental   | N   | N   | N   |
```

Comment:
Will this feature be enabled on v5.0 by default? @eurekaka

Reply:
can we enable it by default in the master branch?

Reply:
If we enable it by default in master branch, we have to maintain 2 sets of CI jobs? otherwise the code paths of disabling plan cache would not be daily tested.

plan cache has already been enabled for tidb-test in https://github.com/pingcap/tidb-test/pull/1010, but for other test suites, it is not yet.

Reply:
yes.

Reply:
Should we move this to a separate issue in `pingcap/tidb`?

Reply:
Sure, since it's not related to this PR.

Reply:
See https://github.com/pingcap/tidb/issues/20306

Reply:
how about marking it as experimental since we haven't enabled it by default at present?